### DISCUSSION TITLE
How do I recreate the current homepage of the website?

### DISCUSSION THREAD

**Discussion by lasofeli** [squidfunk/mkdocs-material#4687] [Q&A] [answered]
It seems that the default homepage of the template when following the documentation is [an emptier version of the actual documentation page](https://squidfunk.github.io/mkdocs-material/creating-your-site/).

Is there a way to make it so that it's similar to the current home page of the site with the fancy full-screen image?

I've seen #1996, but it seems to be referring to a different style home page from the one the site currently has.

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking. The code for the homepage is solely included in [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/), partly due to licensing restrictions of the hero graphic. I can't make the sources public for everyone, so if you want to have a look at the source, you would need to [become a sponsor](https://squidfunk.github.io/mkdocs-material/insiders/#how-to-become-a-sponsor). The homepage code is an exception, as it's not going to get released in the future for the reasons stated.

Note that the source doesn't make much sense without having a similar hero graphic that is divided up into layers. #1996 was referring to our old homepage, which had a different style, but it gives some guidance on theme extension, so definitely a good read.
